src/Integration.Vsix/Delegates/SonarAnalyzerManager.cs

```diff
@@ -116,11 +116,13 @@ private void RefreshWorkflow(BindingConfiguration configuration)
             switch (configuration?.Mode)
             {
                 case SonarLintMode.Standalone:
+                    this.logger.WriteLine(Resources.Strings.AnalyzerManager_InStandaloneMode);
```

Comment:
Changing the package to be asynchronous affects when we start monitoring whether the open solution is in connected more or not. I added some logging to make it easier to see when monitoring has started (although it would be very useful to finally implement the ticket about supporting different levels of logging so we have a verbose mode).